### ISSUE TITLE
[FEATURE 💡] Add method to unsave posts

### ISSUE THREAD

**Issue by warmachine028** [warmachine028/euphoria#28]
## Description:

- Currently we can only save posts
- We need to write a query so that we can also unsave posts from a user's profile.

**Comment by abdulrehmanjaved-1** [comment]
unsave means posts should be deleted or just make them private.

**Comment by warmachine028** [comment]
- when we save a post it is reflected in the UserProfile 
- the unsave option would remove it from the userProfile
same feature like, like and unlike

**Comment by abdulrehmanjaved-1** [comment]
okay assign me

**Comment by warmachine028** [comment]
See if you can work on it, but you will need to work with sanity to create the Query in the backend

**Comment by abdulrehmanjaved-1** [comment]
Okay first let me check

**Comment by warmachine028** [comment]
@abdulrehmanjaved-1 have you made any progress?

**Comment by abdulrehmanjaved-1** [comment]
Hey I had work with sanity long ago. Can you help how to setup sanity server studio in this project.

**Comment by warmachine028** [comment]
Yeah, it's not your fault. This repo really needs some documentation first. 
I'll see what I can manage for you

**Comment by abdulrehmanjaved-1** [comment]
Yea that will help.

**Comment by DhruvGahoi** [comment]
@warmachine028 As of now when I clicks on the save post, it just shows saving and never saves the post

**Comment by warmachine028** [comment]
Is it not showing in the profile? The saved pin. If not then it's a bug

**Comment by DhruvGahoi** [comment]
@warmachine028 no its now showing either in the saved sections.

![image](https://github.com/user-attachments/assets/f729f131-e6d5-45e5-a962-58131dfcbcd0)
![image](https://github.com/user-attachments/assets/1117c5b4-0916-4b57-830a-0b7ae0b44c8a)

**Comment by warmachine028** [comment]
Then it's a bug

**Comment by DhruvGahoi** [comment]
@warmachine028 should I open a issue with this ??

**Comment by warmachine028** [comment]
Please open a new issue

**Comment by DhruvGahoi** [comment]
@warmachine028 can you tell me how to setup the server side, so I start working on both of the issues

**Comment by warmachine028** [comment]
We have used sanity in the backend. Can you please go through the docs. Will be updating the setup steps soon

**Comment by DhruvGahoi** [comment]
@warmachine028 okay I will go through the docs

**Comment by warmachine028** [comment]
> @warmachine028 can you tell me how to setup the server side, so I start working on both of the issues

Follow [this video](https://youtu.be/1RHDhtbqo94). Can also refer the repo in the video description
